### DISCUSSION TITLE
Vragen en opmerkingen

### DISCUSSION THREAD

**Discussion by coolhva** [coolhva/usg-kpn-ftth#27] [General]
Eventuele problemen mogen gemeld worden bij Issues. Vragen en opmerkingen kunnen hier gesteld worden. Bijvoorbeeld de wens voor een nieuwe handleiding voor een ander onderdeel 😄 .

**Comment by TechWizard80** [comment]
Hallo Henk,
Sorry,
Ik had toch een aantal vragen. Ik ben gewoon even nieuwsgierig en misschien kan je me ook helpen.

Ik zie dingen voorbij komen van IPv6, wil ik gewoon niet aan hebben in mijn netwerk. Waarom is dit en is het ook mogelijk alleen IPv4?

IPv6 firewall instellingen, waar voor is dit?

Ik zie ook een vaste DNS server, volgens mij Cloudflare als ik het goed zie, dat wil ik ook graag zelf bepalen in de USG en op welk VLAN welke DNS Server krijgt. Waarom is dit allemaal gedefinieerd in de config? En kan het hele gedeelte van eth1 niet gewoon uitgelaten worden?

En de IGMP proxy, er word 0.0.0.0/0 gedefinieerd, waarom is dit? Kan het niet specifieker voor IPTV of zoiets? En ik wil ook graag IGMP Proxy gebruiken voor mijn HEOS systeem in huis wat op IoT VLAN draait, hoe kan ik dat definiëren?

Hoop niet dat je het erg vind dat ik vraag, maar ben oprecht nieuwsgieren en wil het toch graag anders hebben.

Dank je wel alvast.

**Reply by TechWizard80** [reply]
Is het trouwens niet mogelijk dat bepaalde dingen met de cloudkey (WebGUI) word geconfigureerd? Dat alleen de WAN en IGMP Proxy word geconfigureerd met de JSON file?

**Reply by coolhva** [reply]
De JSON is nodig omdat de WebGUI niet om kan gaan met de IPTV interface aan de WAN zijde. Daarom worden de instellingen in de JSON en in de WebGUI met elkaar gecombineerd tot de uiteindelijke config.

IPv4 adressen raken op en er komen steeds meer diensten die alleen op IPv6 draaien en ook besturingssystemen die een voorkeur hebben voor IPv6. Kort gezegd heeft **iedereen** IPv6 nodig om het internet te kunnen blijven gebruiken zoals we dat vandaag de dag doen. Je kan IPv6 uit de configuratie halen indien je dit niet wilt gebruiken. De firewall instellingen voor IPv6 laten DHCP requests toe van clients in het netwerk en laten ICMP toe.

DNS voor IPv6 staat standaard op cloudflare, dit kan je er uit halen of aanpassen indien nodig.

Er is maar èèn instantie van de IGMP proxy mogelijk op de USG met èèn upstream interface. (https://help.ui.com/hc/en-us/articles/204961854-EdgeRouter-IGMP-Proxy).

Ik heb de configuratie, zoals hij nu is, zo minimaal mogelijk gehouden maar niets weerhoudt je er van om deze aan te passen naar eigen believen.

**Reply by TechWizard80** [reply]
Ja,  mijn netwerk kennis is helaas niet zo sterk.

Denk je dat ik de HEOS op de IoT VLAN werkend kan krijgen als ik IGMP downstream en snooping aanzet? Of krijg ik dan flooding op dat VLAN?

Ik heb wel het 1 en ander uit gehaald, zodat ik gewoon met de WebGUI dingen kan configureren. Zou je eventueel even naar mijn config willen kijken?


[config.gateway.json.txt](https://github.com/coolhva/usg-kpn-ftth/files/6366744/config.gateway.json.txt)

**Reply by TechWizard80** [reply]
Dank je wel voor je hulp

**Reply by coolhva** [reply]
Ik zie een mengelmoes van configs en allerlei zaken die ik anders heb aangepakt (zoals een 0.0.0.0/0 route over eth0.4, dan blijft alles werken als KPN wat aanpast), welk SH bestand ga je gebruiken of wil je elke firmware update de rfc3442-classless-static-routes zelf terugplaatsen, etc.

Ik verwacht niet dat je alle bovenstaande vragen kan beantwoorden, het is meer bedoeld om inzicht te geven dat mijn oplossing doordacht is en samen een complete oplossing biedt met alle componenten. Het is lastig om een component los te beoordelen.

Wat ik aanraad is om te beginnen met mijn complete oplossing zodat het werkt en daarna gewoon zaken eruit te halen en kijken wanneer het fout gaat. 

Als je een kant en klare oplossing wilt waarin geen werk zit en wat "gewoon" werkt dan heeft de KPN daar de experiabox voor. Wil je geen experiabox maar wel gemak dan kan dat ook met de Fritzbox. Wil je met de USG verder dan kost dat meer tijd en moeite en zal je zelf zaken moeten gaan ervaren. Hulp is wel aanwezig, maar dan op dit topic: https://gathering.tweakers.net/forum/list_messages/1883441/last

**Comment by TechWizard80** [comment]
Ok... Dit wel een dingetje inderdaad.
rfc3442-classless-static-routes, ik moet dan even kijken hoe ik dit kan terug zetten.

Ik hoop zo dat Ubiquiti met iets nieuws komt. Ik las iets over een Next-Gen Gateway, en hoop dat ze een vervanger maken van de USG met meer kracht.

**Comment by mskroese** [comment]
Hallo Henk, 

fantastisch werk! Precies wat ik zocht. Mij viel net op dat tegenwoordig de directory "data/sites/<sitename>" niet meer bestaat. In de windows versie van de Unifi controller. Ik heb er wel een aangemaakt maar ik geloof niet dat dat zo werkt. Hierdoor werkt de oplossing denk ik niet. Enig idee hoe dit op te lossen is?

Mvg, 

Maurits
